packages/react-ui/src/Fields/Checkbox.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { Label } from "./Label";
+import { box, fonts, spacings } from "../theme";
+
+export const InputCheckbox = ({ label, name, id, ...props }) => (
+  <StyledWrapper>
```

Comment:
Does StyledWrapper could be merged with StyledLabel (position: relative) ?

Reply:
Done